e2e-tests/utils.sh

```diff
@@ -132,6 +132,9 @@ function retry_times() {
 
 function check_operator_log_for_errors {
   echo "Checking for operator log errors..."
+  echo "Error checking is temporarily turned off."
```

Comment:
Please add a JIRA reference in a comment here @gaborgsomogyi since it might stay like this on the `release-1.3` branch long term

Reply:
Good point, just added.